sunpy/net/fido_factory.py

```diff
@@ -158,6 +158,24 @@ def response_block_properties(self):
             s.intersection(self.get_response(i).response_block_properties())
         return s
 
+    @property
+    def table(self):
+        """
+        Returns `astopy.table.Table` for all responses present in a specific
+        `~sunpy.net.fido_factory.UnifiedResponse` object. They can then be used
+        to perform key-based indexing of QueryResponses
```

Comment:
```suggestion
        to perform key-based indexing of QueryResponses.
```
Can you link to this as you did for UnifiedResponse?